moveit_setup_assistant/src/widgets/setup_assistant_widget.cpp

```diff
@@ -187,6 +187,7 @@ void SetupAssistantWidget::virtualJointReferenceFrameChanged()
   {
     rviz_manager_->setFixedFrame(QString::fromStdString(config_data_->getRobotModel()->getModelFrame()));
     robot_state_display_->reset();
+    robot_state_display_->setVisible(true);
```

Comment:
Was this just never there, or was it removed somehow in a previous change?

Reply:
It was never there because the RobotStateDisplay was showing the robot by default. But this was changed, I think in #1625.